cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/service/subnetselector/SubnetSelectorStrategyMultiplePreferPrivate.java

```diff
@@ -0,0 +1,50 @@
+package com.sequenceiq.cloudbreak.cloud.aws.service.subnetselector;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.model.CloudSubnet;
+
+@Component
+public class SubnetSelectorStrategyMultiplePreferPrivate extends SubnetSelectorStrategy {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SubnetSelectorStrategyMultiplePreferPrivate.class);
+
+    private static final int MIN_SUBNET_IN_DIFFERENT_AZ = 2;
+
+    private static final int MAX_SUBNET_IN_DIFFERENT_AZ = 3;
```

Comment:
could you move this into `@Values` so we can modify it easily if needed

Reply:
Yes, did it, thanks.